Cargo.toml

```diff
@@ -65,15 +79,47 @@ ibc-core-commitment-types   = { version = "0.56", default-features = false }
 ibc-core-handler-types      = { version = "0.56", default-features = false }
 ibc-client-tendermint-types = { version = "0.56", default-features = false }
 
-alloy             = { version = "0.7", default-features = false }
-alloy-contract    = { version = "0.7", default-features = false }
-alloy-sol-types   = { version = "0.8", default-features = false }
-alloy-primitives  = { version = "0.8", default-features = false }
+alloy                  = { version = "0.7", default-features = false }
+alloy-contract         = { version = "0.7", default-features = false }
+alloy-sol-types        = { version = "0.8", default-features = false }
+alloy-primitives       = { version = "0.8", default-features = false, features = ["serde", "rlp"] }
+alloy-trie             = { version = "0.5", default-features = false, features = ["serde"] }
+alloy-serde            = { version = "0.7", default-features = false }
+alloy-rlp              = { version = "0.3", default-features = false }
+alloy-rpc-types-eth    = { version = "0.7", default-features = false, features = ["serde"] }
+alloy-rpc-types-beacon = { version = "0.7", default-features = false, features = ["ssz"] }
+alloy-rpc-types-engine = { version = "0.7", default-features = false, features = ["ssz"] }
 
 sp1-sdk    = { version = "3.3", default-features = false }
 sp1-zkvm   = { version = "3.3", default-features = false }
 sp1-helper = { version = "3.3", default-features = false }
 
+cosmwasm-schema = { version = "2.1", default-features = false }
+cosmwasm-std    = { version = "2.1", default-features = false }
+cw-storage-plus = { version = "2.0", default-features = false }
+
+# The dependencies below are maintained by Sigma Prime (for use in Lighthouse (and the broader Ethereum ecosystem))
+ethereum_ssz        = { version = "0.8", default-features = false }
+ethereum_ssz_derive = { version = "0.8", default-features = false }
+tree_hash           = { path = "packages/tree_hash", version = "0.8", default-features = false }
```

Comment:
this is a local crate so should move to the top?

Reply:
Good point. Added it late last night as a fork to fix cosmwasm build